### DISCUSSION TITLE
Welcome to storm-engine Discussions!

### DISCUSSION THREAD

**Discussion by np-c0mp1ete** [storm-devs/storm-engine#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Cybertwip** [comment]
Hi, I'm Victor, better known for contributions as Greentwip, I've ported Python 3.4 and Python 2.7 to Xbox One along with PyGame for certain remakes (one based on Nintendo's Fire Emblem), and its on my plans to port Ren'Py to the console. I started with this about video games 9 years ago, working at Gameloft as a mobile developer specialized in porting. Recently I've been working with the Sonic 2 HD team and my contributions have been gameplay, shader writing and porting.

I would like to add into the challenge of implementing the bgfx rendering backend for Xbox One, as of, its plain DirectX, I believe with bgfx we have a super similar approach because of the low level texture and frame buffer creation, it looks mostly like a DirectX to OpenGL port. 

I'm super glad to acquaint with you, let's get some fun!

**Reply by np-c0mp1ete** [reply]
@VictorJL wow, great to have you on board! Please join our discord channel and let's start some technical discussions regarding your future contributions ;)

**Comment by shub124** [comment]
Hi, I am Shubham. I would like to start contributing to storm-engine. However, as somebody who is a first timer, is there some documentation to understand the codebase?

**Reply by np-c0mp1ete** [reply]
Hi Shubham! No, unfortunately there is no proper engine documentation. We have a task to organize the documentation better (https://github.com/storm-devs/storm-engine/projects/4#card-58081815). There are several links to some guides about the engine, but I'm not sure they will be useful for understanding the codebase.

Learning the codebase by doing some simple tasks from our backlog (https://github.com/storm-devs/storm-engine/projects) is probably the best way to learn it.
Don't hesitate to ask questions on the Discord channel, we'll be glad to help you :)
